website/source/layouts/guides.erb

```diff
@@ -50,6 +50,10 @@
         </ul>
       </li>
 
+      <li<%= sidebar_current("guides-tls") %>>
+        <a href="/guides/tls.html">Configuring TLS</a>
```

Comment:
Better SEO is my optimization. In the future, it would be very easy to move securing-nomad.html to securing-nomad/index.html and then have securing-nomad/traffic.html and securing-nomad/production-hardening.html, etc